src/main/java/com/synopsys/integration/jenkins/scan/global/Utility.java

```diff
@@ -13,8 +15,98 @@ public static FilePath createTempDir(String directoryName) throws IOException, I
         return tempFilePath;
     }
 
-    public static void cleanupTempDir(FilePath tempDir) throws IOException, InterruptedException {
-        tempDir.delete();
+    public static void cleanupTempDir(FilePath tempDir){
+        try {
+            if (tempDir.exists()) {
+                tempDir.deleteRecursive();
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
     }
 
+    public static void deleteDirectory(File directory) {
+        File[] files = directory.listFiles();
+        if (files != null) {
+            for (File file : files) {
+                if (file.isDirectory()) {
+                    deleteDirectory(file);
+                } else {
+                    file.delete();
+                }
+            }
+        }
+        directory.delete();
+    }
+
+    public static void copyRepository(String targetDirectory, String pluginWorkspaceDirectory) {
```

Comment:
We can alter the parameters place. 
`copyRepository(pluginWorkspaceDirectory, targetDirectory)` is easily understandable like `FileUtils.copyDirectory(workspaceDirectory, targetDir)`